### DISCUSSION TITLE
Contributors Open to <model-viewer> Projects?

### DISCUSSION THREAD

**Discussion by shawn-wm** [google/model-viewer#2917] [General]
I didn't see any mention in the Code of Conduct around third-party project support requests, so I apologize in advance if I'm overstepping.

I am looking for support in implementing a project utilizing model-viewer, and am wondering if there are any contributors here that would be open to that, and how best we might facilitate it.

**Comment by elalish** [comment]
We have no problem with people using  our Github  as a forum for advertising their services or requesting them. If it becomes popular enough, we'll probably make a category for it. You're all our users after all 😁.
